apps/semantic-cache/types/index.ts

```diff
@@ -0,0 +1,29 @@
+import type { Ai } from "@cloudflare/ai";
+import type { KVNamespace, VectorizeIndex } from "@cloudflare/workers-types";
+
+export type Response = {
+  id: string;
+  content: string;
+};
+
+export type AnalyticsEvent = {
```

Comment:
probably better to generate this from the zod schema, so they don't desync

Reply:
Done